src/openlifu/db/database.py

```diff
@@ -71,6 +71,28 @@ def write_protocol(self, protocol: Protocol, on_conflict: OnConflictOpts = OnCon
 
         self.logger.info(f"Added Sonication Protocol with ID {protocol_id} to the database.")
 
+    def delete_protocol(self, protocol_id: str, on_conflict: OnConflictOpts = OnConflictOpts.ERROR):
+        # Check if the sonication protocol ID already exists in the database
+        protocol_ids = self.get_protocol_ids()
+
+        if protocol_id not in protocol_ids:
+            if on_conflict == OnConflictOpts.ERROR:
+                raise ValueError(f"Protocol ID {protocol_id} does not exist in the database.")
+            elif on_conflict == OnConflictOpts.SKIP:
+                self.logger.info(f"Cannot delete protocol ID {protocol_id} as it does not exist in the database.")
+                return
+            else:
+                raise ValueError("Invalid 'on_conflict' option. Use 'error', 'overwrite', or 'skip'.")
```

Comment:
```py
raise ValueError("Invalid 'on_conflict' option. Use 'error', 'overwrite', or 'skip'.")
```

also "overwrite" wouldn't make sense here

in fact this whole message doesn't make sense-- it is there in other places as a holder from back when we had strings instead of the `OnConflictOpts` enum. better to just say `"Invalid 'on_conflict' option.` and leave it at that